src/pages/Auth.tsx

```
@@ -1,93 +1,493 @@
+    if (!email || !password) {
+      setFormError("Por favor, introduce tu email y contraseña");
+      return false;
+    }
+
+    setFormError(null);
+    setIsSubmitting(true);
+
+    try {
+      await login(email, password);
+      return true;
+    } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)